expo/src/api/GnoNativeApi.ts

```diff
@@ -156,60 +170,88 @@ export class GnoNativeApi implements GnoKeyApi, GoBridgeInterface {
   }
 
   async generateRecoveryPhrase() {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.generateRecoveryPhrase(new GenerateRecoveryPhraseRequest());
     return response.phrase;
   }
 
   async hasKeyByName(name: string) {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.hasKeyByName(new HasKeyByNameRequest({ name }));
     return response.has;
   }
 
   async hasKeyByAddress(address: Uint8Array) {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.hasKeyByAddress(new HasKeyByAddressRequest({ address }));
     return response.has;
   }
 
   async hasKeyByNameOrAddress(nameOrBech32: string) {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.hasKeyByNameOrAddress(
       new HasKeyByNameOrAddressRequest({ nameOrBech32 }),
     );
     return response.has;
   }
 
   async getKeyInfoByName(name: string): Promise<KeyInfo | undefined> {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.getKeyInfoByName(new GetKeyInfoByNameRequest({ name }));
     return response.key;
   }
 
   async getKeyInfoByAddress(address: Uint8Array): Promise<KeyInfo | undefined> {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.getKeyInfoByAddress(new GetKeyInfoByAddressRequest({ address }));
     return response.key;
   }
 
   async getKeyInfoByNameOrAddress(nameOrBech32: string): Promise<KeyInfo | undefined> {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.getKeyInfoByNameOrAddress(
       new GetKeyInfoByNameOrAddressRequest({ nameOrBech32 }),
     );
     return response.key;
   }
 
   async listKeyInfo(): Promise<KeyInfo[]> {
-    const client = await this.#getClient();
+    const client = this.#getClient();
     const response = await client.listKeyInfo(new ListKeyInfoRequest());
     return response.keys;
   }
 
+  async makeCallTx(
+    packagePath: string,
+    fnc: string,
+    args: string[],
+    gasFee: string,
+    gasWanted: bigint,
```

Comment:
These parameters should be the same as `call`. In `call`, gasWanted is a `number` and is [converted to bigint](https://github.com/gnolang/gnonative/blob/ecdd187838f1e60acd036daadd07c4414ba6ffeb/expo/src/api/GnoNativeApi.ts#L334).

Reply:
I asked me the question. The best would be to do a changing break and also convert gasWanted as a bigint in the call function. Otherwise, we will limit the front end to put big values.
What do you think?

Reply:
In the Go code, these are normal integers. I don't know why the buf generator makes them bigint. I think we should as Iuri what he thinks.

Reply:
They are int64 which is the equivalent of bigint in typescript.
https://github.com/gnolang/gnonative/blob/main/api/gnonativetypes/gnonativetypes.go#L251

Reply:
Makes sense to keep as bigint because converting them to number could...

> Be careful coercing values back and forth, however, as the precision of a BigInt value may be lost when it is coerced to a Number value

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt